### PR TITLE
WIP: release notes 9.23.0

### PR DESCRIPTION
**Pull request by mritense**
-

### CODE REVIEW

File: release-notes/major9/09.23.0/valtimo-backend-libraries.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Backend libraries 9.23.0 (RC)
+
+## New Features
+
+The following features were added:
+
+* **Allow external fields to have a nested formfield.key structure**
```
Comment (Klaas-Ritense) [inline release-notes/major9/09.23.0/valtimo-backend-libraries.md:7]: This belongs in frontend

File: release-notes/major9/09.23.0/valtimo-backend-libraries.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+  X was deprecated and is replaced with Y.
+
+Instructions on how to migrate to this version of Valtimo can be found [here](migration.md).
+
+## Known issues
+
+This version has the following known issues:
+
+* **Issue1**
```
Comment (marijnritense) [inline release-notes/major9/09.23.0/valtimo-backend-libraries.md:75]: This should be removed

File: release-notes/major9/09.23.0/migration.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Migration
+
+This page describes how to update Valtimo from the previous version to the current.
+
+* **Breaking change 1/Deprecation 1**
```
Comment (marijnritense) [inline release-notes/major9/09.23.0/migration.md:5]: Migration notes should be removed if empty

File: SUMMARY.md [hunk 4]
```diff
@@ -107,11 +107,15 @@
   * [User interface](reference/user-interface/user-interface.md)
     * [Design philosophy](reference/user-interface/design-philosophy.md)
   * [Troubleshoot](reference/troubleshoot.md)
-* [Release notes](release-notes/release-notes.md)
-  * Major 9
-    * 9.22.0/5.11.0 (RC)
-      * [Valtimo backend libraries (9.22.0 RC)](release-notes/major9/09.22.0/valtimo-backend-libraries.md)
-      * [Valtimo frontend libraries (5.11.0 RC)](release-notes/major9/09.22.0/valtimo-frontend-libraries.md)
+* Release notes
```
Comment (marijnritense) [inline SUMMARY.md:110]: Should link to the release notes page still

File: SUMMARY.md [hunk 5]
```diff
@@ -107,11 +107,15 @@
...
+  * 9.x.x
+    * 9.23.0/5.12.0 (RC)
+      * [Valtimo backend libraries (9.23.0 RC)](release-notes/major9/09.23.0/valtimo-backend-libraries.md)
+      * [Valtimo frontend libraries (5.12.0 RC)](release-notes/major9/09.23.0/valtimo-frontend-libraries.md)
+      * [Migration](release-notes/major9/09.23.0/migration.md)
```
Comment (marijnritense) [inline SUMMARY.md:115]: This link can be removed

**Review by marijnritense — APPROVED** [review]

**Pull request merged**